docs/content/en/docs/concepts/eksapackages.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "EKS Anywhere curated packages"
+linkTitle: "EKS Anywhere curated packages"
+weight: 4
+date: 2022-04-12
+description: >
+  Explanation of the process of managing curated packages in an EKS Anywhere cluster
+---
+
+EKS Anywhere Curated Packages is a management system for installation, configuration and maintenance of additional components for your Kubernetes applications. Examples of these components may include Container Registry, Ingress, and LoadBalancer, etc.
+
+## Overview
+The major components of EKS Anywhere Curated Packages are the package controller, the package build artifacts and the command line interface. The package controller will run in a pod in an EKS Anywhere cluster. The package controller will manage the lifecycle of packages tested and maintained by EKS Anywhere.
```

Comment:
Three items are listed here and we only talk about one in this paragraph and two in the next paragraph and the next two sections are `##` instead of `###`

Reply:
Maybe get rid of that `## Overview` and add a paragraph about the cli